rpc/rpc-wrappers.go

```diff
@@ -64,108 +64,99 @@ type certificateRequest struct {
 	Key jose.JsonWebKey
 }
 
-func NewRegistrationAuthorityServer(serverQueue string, channel *amqp.Channel, impl core.RegistrationAuthority) (rpc *AmqpRpcServer, err error) {
-	rpc = NewAmqpRpcServer(serverQueue, channel)
+func NewRegistrationAuthorityServer(serverQueue string, channel *amqp.Channel, impl core.RegistrationAuthority) (rpc *AmqpRPCServer, err error) {
+	rpc = NewAmqpRPCServer(serverQueue, channel)
 
-	rpc.Handle(MethodNewAuthorization, func(req []byte) (response []byte) {
+	rpc.Handle(MethodNewAuthorization, func(req []byte) []byte {
 		var ar authorizationRequest
-		err := json.Unmarshal(req, &ar)
-		if err != nil {
-			return
+		if err := json.Unmarshal(req, &ar); err != nil {
+			return nil
 		}
 
 		authz, err := impl.NewAuthorization(ar.Authz, ar.Key)
 		if err != nil {
-			return
+			return nil
```

Comment:
Not enough arguments to return, since this method returns two args. Here and elsewhere: this is what's causing Travis to fail.